src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -39,7 +41,7 @@ public void loadSomeData()
 					final Gson gson = new GsonBuilder().create();
 					final Map venue = gson.fromJson(content, Map.class);
 
-					final List<Map> placeCategories = (List<Map>) venue.get("categories");
+					placeCategories = (List<Map>) venue.get("categories");
 					scanner.close();
 				}
 				catch (final FileNotFoundException e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1166)